node-definitions/sonic/README.md

```diff
@@ -0,0 +1,13 @@
+This directory contains the node definition for SONiC 
+
+
+# Notes
+
+default username and password are admin/YourPaSsWoRd
+mgmt interface defaults to dhcp
+
+download community image here : https://sonic.software
```

Comment:
Also, please put a URL to the documentation for the node.

Reply:
Are you referring user config guide ?